cfme/utils/template/rhopenshift.py

```diff
@@ -162,7 +156,13 @@ def run(self):
             return False
 
         for template in self.templates:
-            self.track_template(self.stream, self.provider_key, template['name'],
-                                custom_data={'TAGS': self.tags})
+            result = self.track_template(
+                stream=self.stream,
+                provider_key=self.provider_key,
+                template_name=template['name'],
+                custom_data={'TAGS': self.tags}
+            )
+            if result is False:
```

Comment:
Thanks - track_template isn't actually returning anything, so I added return there.

I don't actually want to use `bool(result)` here, because there are three states - None means it already existed, which would trigger this returning False. If the record already existed, I think that's a clean exit state, and we can return True.